libs/ptg-ui-web-components/src/components/ptg-accordion/ptg-accordion.tsx

```diff
@@ -24,23 +25,33 @@ export class PtgAccordion {
     return (
       <div>
         <button
-          class="accordion"
+          class={`accordion ${this.toggle ? 'active' : ''}`}
           style={{
             width: this.width,
             backgroundColor: this.color,
           }}
-          onClick={() => this.toggleComponent()}
-        >
+          onClick={() => this.toggleComponent()}>
           {this.label}
-          {this.toggle ? <span>&#9650;</span> : <span>&#9660;</span>}
+          {this.toggle ? (
+            <span class="arrow down"></span>
+          ) : (
+            <span class="arrow"></span>
+          )}
         </button>
         <div
           class={`content-box ${this.toggle ? 'open' : 'close'}`}
-          style={{ width: this.width }}
-        >
-          <p>{this.description}</p>
+          style={{ width: this.width }}>
```

Comment:
why we need to give width here ? it should have 100% width. whoever consuming it can handle width from their container parent.